### PR TITLE
frontend: add printTemplateInformationsheet

### LINKED ISSUE #70: frontend: add information sheet
Create a printeable information sheet which provides:

- instructions for the voting

- list of all voteable seminars

### PR DESCRIPTION
**Pull request by gilgda**
Add the print template for the informationsheet which should be handed out before the vote.
Closes #70

### CODE REVIEW

File: frontend/libs/printTemplateInformationsheet.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  <body>
+    <div class="page" style="width: 100%;">
+      <img style="vertical-align: middle" src="../src/assets/logo.png" width="100%" class="logo">
+      <div style="position: relative; left: 50%;">
+        <span style="display: table-cell; width: 100%; margin: 0 auto; font-weight: bold; font-size: 1.17em; text-align: center;">Wahlsystem des Werdenfels-Gymnasium</span>
+      </div>
+      <div style="position: relative; margin-left: 50px; margin-right: 50px; text-align: justify; ">
+        <h4 style="margin-top: 40px; margin-bottom: 5px; text-align: left;">Liebe Schülerinnen und Schüler, </h4>
+        <p style="margin-top: 5px; text-align:left;">
+          Seit Kurzem verwendet das Werdenfels-Gymnasium ein neues, digitales System zur Durchführung von Wahlen wie beispielsweise der Seminare.<br>
```
Comment (Lordnoname) [inline frontend/libs/printTemplateInformationsheet.html]: `Seit` lower case

Comment (Lordnoname) [inline frontend/libs/printTemplateInformationsheet.html]: IMO: Replace `Seminare` with `Seminarwahl`

File: frontend/libs/printTemplateInformationsheet.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          Hierzu nun eine kurze Einführung mit hilfreichen Informationen zu der Wahl Ihrer Seminare in der Oberstufe.
+        </p>
+        <p>
+          <h4 style="margin-bottom: 5px;"><i>Allgemeiner Wahlvorgang</i></h4>
+          Die Wahl von W- und P-Seminar findet wie früher mittels einer priorisierten Abgabe von Stimmen für Seminare statt.
```
Comment (Lordnoname) [inline frontend/libs/printTemplateInformationsheet.html]: `Wahl von W- und P-Seminar` -> `Wahl des W- und P-Seminars`

File: frontend/libs/printTemplateInformationsheet.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+          Das heißt, dass Sie je drei Seminare auswählen, die Sie in Erst-, Zweit- und Drittwahl einteilen.
+          Dabei stellt Ihre Erstwahl das am Meisten gewünschte Seminar dar und die Zweit- beziehungsweise Drittwahl Alternativen dazu.
+          Das Seminar, welches Sie am Ende belegen ist eines der drei gewählten Seminare und entspricht je nach Belegung der einzelnen Seminare idealerweise Ihrer Erstwahl.<br>
```
Comment (Lordnoname) [inline frontend/libs/printTemplateInformationsheet.html]: You can get an alternative seminar, if the circumstances are really bad.

File: frontend/libs/printTemplateInformationsheet.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            <li>Nachdem Sie nun wieder auf dem Startbildschirm sind können Sie:
+              <ul>
+                <li>eine weitere Wahl tätigen,</li>
+                <li>die bisher getätigten Wahlen nochmals ändern (siehe 2.)</li>
+              </ul>
+            </li>
+            <li>Melden Sie sich durch Klicken auf "Logout" am linken unteren Bildschirmrand ab. Das Programm kann jetzt beendet werden.</li>
+          </ol>
+          Die getätigte Wahl wird im System gespeichert und kann noch bis zum Abgabedatum geändert werden.<br>
+          Falls weiterhin Fragen bestehen können Sie sich an die Oberstufenkoordinatoren Frau Berger und Herr Bockhorni wenden.<br>
```
Comment (Lordnoname) [inline frontend/libs/printTemplateInformationsheet.html]: I would not name the OSKs because they change from time to time...

**Comment by Lordnoname** [comment]
This sheet should be integrated in the vote sheet of each student.

**Comment by gilgda** [comment]
I would not integrate it into the vote sheet as it should be handed out to every student as information **before** they vote to know how to vote.

File: frontend/libs/printTemplateInformationsheet.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            <li>Nachdem Sie nun wieder auf dem Startbildschirm sind können Sie:
+              <ul>
+                <li>eine weitere Wahl tätigen,</li>
+                <li>die bisher getätigten Wahlen nochmals ändern (siehe 2.)</li>
+              </ul>
+            </li>
+            <li>Melden Sie sich durch Klicken auf "Logout" am linken unteren Bildschirmrand ab. Das Programm kann jetzt beendet werden.</li>
+          </ol>
+          Die getätigte Wahl wird im System gespeichert und kann noch bis zum Abgabedatum geändert werden.<br>
+          Falls weiterhin Fragen bestehen können Sie sich an die Oberstufenkoordinatoren.<br>
```
Comment (Lordnoname) [inline frontend/libs/printTemplateInformationsheet.html]: sentence incomplete

**Pull request merged**